web/client/epics/mapexport.js

```diff
@@ -38,6 +38,7 @@ const saveMap = (state, addBbox = false) => {
         groupsSelector(state),
         backgroundListSelector(state),
         textSearchConfigSelector(state),
+        bookmarkSearchConfigSelector(state),
```

Comment:
will this change need a dedicated unit test?

Reply:
I don't see any specific scenario requirements, hence not required. And it is covered as part of MapUtils